modules/sql-engine/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -761,6 +761,10 @@ SqlNode SqlAlterZone() :
         { s.add(this); } optionList = AlterZoneOptions() {
           return new IgniteSqlAlterZoneSet(s.end(this), zoneId, optionList, ifExists);
         }
+        |
+        { s.add(this); } optionList = ZoneOptionsList() {
```

Comment:
I think we should also allow simplified form without parenthesis for alteration of a single option: `ALTER ZONE my_zone SET replicas 4`

Reply:
changed